src/java/com/unifina/signalpath/custom/CompilationErrorMessage.java

```diff
@@ -9,15 +9,15 @@
 	List<Map> errors = new ArrayList<>();
 	
 	public CompilationErrorMessage() {
```

Comment:
The stuff in this class should go to `JavaCompilerErrorMessage#toMap()` and this class should be deleted.